submissions/AndriiZahumnyi/a-tiny-js-world-pre-oop/index.js

```diff
@@ -0,0 +1,79 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/AndriiZahumnyi/a-tiny-JS-world
+   Web app: https://andriizahumnyi.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const man = {
+   species: 'human',
+   name: 'John',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi! How are you?',
+   friendlyTo: ['Toby', 'Tereza', 'Tom']
+};
+
+const woman = {
+   species: 'human',
+   name: 'Tereza',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'I`m fine, thanks!',
+   friendlyTo: ['Tom']
+};
+
+const dog = {
+   species: 'dog',
+   name: 'Toby',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'woof-woof!',
+   friendlyTo: ['John', 'Tereza']
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Tom',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'meow-meow!',
+   friendlyTo: ['Tereza']
+};
+
+function printObject(object) {
+   print(Object.values(object).join('; '));
```

Comment:
Order of values in an object is not guaranteed by system design.
Please use explicit list of properties.

This function is responsible for two different activities: form a string from an object properties AND perform output.

This violates SRP principle.

What would you change if we wanted to use object presentation string for multiple purposes?

Reply:
> @AndriiZahumnyi good job.
> 
> Please check comments below and this [message in tg chat](https://t.me/c/1382428271/27203)

Thank you for review!
I`ll fix all these things and report!